core/db/entities.py

```diff
@@ -148,7 +148,7 @@ class Procurement(db.Model):
     project_id = sqla.Column(sqla.String(), sqla.ForeignKey("project_dim.id"), nullable=False)
     start_date = sqla.Column(sqla.DateTime(), nullable=False)
     end_date = sqla.Column(sqla.DateTime(), nullable=False)
-    status = sqla.Column(sqla.Enum(const.StatusEnum, name="procurement_plan_status"), nullable=False)
```

Comment:
Good spot. Is this definitely correct now? According to the Enum Spreadsheet this is now right, but the original data model spreadsheet had 2 different status types in it for Procurement. Was this superseded when the Enum definitions were created (I'm just checking this isn't a missed field)?

Reply:
You're right, nice spot! Glad you checked it haha. This didn't fall over during run time because I had a mistake where I was overwriting 'status' with 'procurement_status' in the ingest PR

Reply:
So just deleting the original "status" data with "procuement_status" basically

Reply:
fixed this 👍